nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/TestDataTypeUtils.java

```diff
@@ -168,6 +168,21 @@ public void testConvertRecordFieldToObject() {
 
     }
 
+    @Test
+    public void testToArray() {
+        final ArrayList<String> arrayList = new ArrayList<>();
+        arrayList.add("Seven");
```

Comment:
This could also be reduced to `final list<String> arrayList = Arrays.asList("Seven", "Eleven", "Thirteen");`

Reply:
I have updated the PR.  Thanks!